### ISSUE TITLE
execvp: No such file or directory

### ISSUE THREAD

**Issue by swcraftsmen** [ysbaddaden/prax.cr#59]
I am seeing this error message when I ran "curl http://myapp.dev'
```
<body>
  <div class="page">
    <h1 class="err">Error starting application</h1>
<!--h2>Your Rack app raised an exception when Pow tried to run it.</h2-->

<section>
  <pre>execvp: No such file or directory (Errno)
0x451cf8: ??? at /opt/crystal/src/process.cr 381:7
0x44f0ba: ??? at /vagrant/src/prax/application/spawner.cr 52:13
0x41b828: ??? at /opt/crystal/src/fiber.cr 114:3
0x0: ??? at ??
</pre>
</section>
```

**Comment by ysbaddaden** [comment]
Here is the failing call:
https://github.com/ysbaddaden/prax.cr/blob/master/src/prax/application/spawner.cr#L77

Maybe it can't find `prax-rc` (which should along `bin/prax`)?

**Comment by zachary-huang** [comment]
I don't see prax-rc inside the `opt/prax/bin/` and I only see `prax` and `prax-binary`

**Comment by zachary-huang** [comment]
I manually copy the file to the `/opt/prax/bin`  and I still seeing this error

**Comment by ysbaddaden** [comment]
Should have been fixed by b6b299e96371785cc3b25c44ffbb94db6bab4cea.
